examples/ota-provider-app/esp32/main/BdxOtaSender.cpp

```diff
@@ -82,12 +82,24 @@ void BdxOtaSender::HandleTransferSessionOutput(TransferSession::OutputEvent & ev
             // end of the transfer.
             sendFlags.Set(chip::Messaging::SendMessageFlags::kExpectResponse);
         }
-        VerifyOrReturn(mExchangeCtx != nullptr, ChipLogError(BDX, "%s: mExchangeCtx is null", __FUNCTION__));
+        if (mExchangeCtx == nullptr)
```

Comment:
nit: I know you're dropping the log message but could just do:
```suggestion
        VerifyOrReturn(mExchangeCtx != nullptr);
```